src/components/ActionButton.tsx

```diff
@@ -88,7 +88,16 @@ export default function ActionButton({
   wrapperProps,
   ...rest
 }: ActionButtonProps) {
-  const textColor = useMemo(() => (color === 'accent' ? 'onAccent' : 'currentColor'), [color])
+  const textColor = useMemo(() => {
+    switch (color) {
+      case 'accent':
```

Comment:
This feels like a failing of the design system. How come `accentSoft` does not get an `onAccentSoft` color? Should `onAccentSoft` alias to `accent`?

Reply:
Could you bring this up with Design? It doesn't need to block this PR, but I would like to make sure it's brought up.

Reply:
yeah i'll start a discussion with design about this ... i'm also confused why the widget has a much smaller color set. if it's not intentional, then i'd like to copy over the color definitions from `interface` since the new widget designs are using the same system